src/components/studies/app-design/GeneralContactAndSupportSection.tsx

```diff
@@ -0,0 +1,254 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core'
+import Subsection from './Subsection'
+import { FormControl, FormGroup, Box, FormHelperText } from '@material-ui/core'
+import { StudyAppDesign } from '../../../types/types'
+import { playfairDisplayFont, poppinsFont } from '../../../style/theme'
+import {
+  SimpleTextInput,
+  SimpleTextLabel,
+} from '../../widgets/StyledComponents'
+import { AppDesignUpdateTypes } from './AppDesign'
+import clsx from 'clsx'
+import { isInvalidPhone, isValidEmail } from '../../../helpers/utility'
+import { makePhone } from '../../../helpers/utility'
+
+const useStyles = makeStyles(theme => ({
+  formFields: {
+    fontFamily: poppinsFont,
+    fontSize: '14px',
+    marginBottom: '24px',
+    '& .MuiFormControl-root:not(:last-child)': {
+      marginBottom: '16px',
+    },
+  },
+  firstFormElement: {
+    marginTop: theme.spacing(2.5),
+  },
+  informationRowStyle: {
```

Comment:
I agree with the formFields and informationRowStyle style classes. principleInvestigatorsParagraph actually only should appear in 1 of the components but currently appears in 2 because I forgot to take it out of the `GeneralContactAndSupportSection.tsx`component. I will make two wrapper components to reduce redundancy for the `informationRowStyle` and `formFields` style classes.